__tests__/app.test.js

```diff
@@ -78,6 +78,155 @@ describe("/api/reviews", () => {
           expect(body.msg).toBe("Route not found");
         });
     });
+
+
+    describe("queries", () => {
+      test("accepts category query", () => {
+        return request(app)
+          .get("/api/reviews?category=dexterity")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.reviews).toBeInstanceOf(Array);
+            expect(body.reviews).toHaveLength(1);
+
+            expect(body.reviews[0]).toMatchObject({
```

Comment:
Would do a forEach here rather than just check the first element and ensure they all have a category of dexterity